content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -36,38 +36,37 @@ ENABLE_SENSU_SANDBOX_PORT_FORWARDING=1 vagrant up
 {{< /highlight >}}
 
 The Learn Sensu sandbox is a CentOS 7 virtual machine pre-installed with Sensu, InfluxDB, and Grafana.
-It is intended for use as a learning tool; we do not recommend this tool as part of a production installation.
-To install Sensu in production, please see the [installation guide](../../installation/install-sensu).
-The sandbox startup process takes about five minutes.
+It's intended for you to use as a learning tool--we do not recommend using it in a production installation. To install Sensu in production, use the [installation guide](../../installation/install-sensu) instead.
```

Comment:
minor nit (and maybe the format isn't translating here), but could we please use an em dash with a space on either end? e.g., "...a learning tool — we do not recommend..."